app/services/publisher_notifier_unverified.rb

```diff
@@ -0,0 +1,67 @@
+class PublisherNotifierUnverified < BaseService
+  attr_reader :notification_params, :notification_type
+
+  PUBLISHER_TYPES = %w(youtube, domain)
```

Comment:
let's remove the youtube references and require that the type be `domain` for now.  we can add a comment up top noting TODO: youtube